content/docs/1.3.0/getting-started/quick-start.md

```diff
@@ -19,20 +19,32 @@ brew tap eclipse-iofog/iofogctl
 brew install iofogctl@1.3
 ```
 
+#### Install iofogctl on Windows
+
+The Windows binary can be downloaded from https://storage.googleapis.com/iofogctl/win/1.3/iofogctl.exe.
+
+##### Prepare Windows
+
+In order to use `iofogctl` to deploy ioFog locally (as this quick-start guide does), we will need to follow a few steps to enable Docker to be able to run Linux containers within Windows:
+
+* Install [docker desktop for windows](https://download.docker.com/win/stable/Docker%20Desktop%20Installer.exe)
+* Enable Hyper-V in Powershell `Install-WindowsFeature -Name Hyper-V -IncludeManagementTools -Restart`
+* Ensure that docker is running with [Linux containers mode](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers)
+
 #### Install iofogctl on Linux
 
 The Debian package can be installed like so:
 
 ```bash
 curl https://packagecloud.io/install/repositories/iofog/iofogctl/script.deb.sh | sudo bash
-sudo apt-get install iofogctl=1.3.1
+sudo apt-get install iofogctl=1.3.2
```

Comment:
Don't we want to go for `apt-get install iofogctl` and put the version once we reach 1.4.

So that the lastest doc always installs the latest version ?